packages/creamy_field/example/lib/main.dart

```diff
@@ -51,20 +51,11 @@ class _MyEditorAppState extends State<MyEditorApp> {
           FlatButton(
             child: Text('Add tab'),
             onPressed: () {
-              // Adds a tab at the selection's base base-offet
```

Comment:
This comment has a short usage-description of `addTab` property in CreamyEditingController. I believe this example code needs this comment to make its usage more clear.